recipe/meta.yaml

```diff
@@ -1,30 +1,32 @@
 {% set name = "brightway2" %}
-{% set version = "2.4.2" %}
+{% set version = "2.4.3" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 8e741b0e9fe504bc740edad800e17eec9907ed53c181ddab3038801ed8caee80
+  sha256: e9521fbb6368689e784d7f5454e89739db3a7d7ea6c64a6089d9e5f45f49a4a7
 
 build:
-  noarch: python
```

Comment:
Why do changed it to a non-noarch package?

Reply:
Because it [failed to build](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=664213&view=logs&jobId=656edd35-690f-5c53-9ba3-09c10d0bea97&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d). It uses `bw2calc` as a dependency, and this is OS-specific; in theory this shouldn't matter for downstream packages, but this fixed it.

Maybe I just triggered it too quickly after the new `bw2calc` build? I guess we could change it, but is there some reason we can't leave it, as it seems to be working?

Reply:
> Maybe I just triggered it too quickly after the new `bw2calc` build?

Could be, I just updated the requirements in #5, lets see if it passes there as a noarch-package.

>  I guess we could change it, but is there some reason we can't leave it, as it seems to be working?

We should prefer a noarch package: https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-builds

> Noarch packages are packages that are not architecture specific and therefore only have to be built once.
>
> Declaring these packages as noarch in the build section of the meta.yaml, reduces shared CI resources. Therefore all packages that qualify to be noarch packages should be declared as such.

Reply:
Build just passed, I will merge #5 i favor of this.